docs/conf.py

```diff
@@ -30,20 +30,32 @@
 import configparser
 from datetime import datetime
 
+from pkg_resources import get_distribution
 from packaging.requirements import Requirement
 
 try:
     import importlib.metadata as importlib_metadata
 except ImportError:
     import importlib_metadata
 
-try:
-    from sphinx_astropy.conf.v1 import *  # noqa
-except ImportError:
-    print('ERROR: the documentation requires the sphinx-astropy package to be installed')
+# -- Check for missing dependencies -------------------------------------------
+doc_requires = get_distribution("astropy").requires(extras=("docs",))
```

Comment:
Wait a min... I thought we are using importlib now.

https://github.com/astropy/astropy/blob/75075236dc543255e82ea3609ff8ab2449179d46/docs/conf.py#L107

Reply:
Yeah indeed, we should use `importlib.metadata` instead (#11091).

Reply:
Done.